src/appengine/handlers/cron/project_setup.py

```diff
@@ -819,7 +820,7 @@ def _sync_job(self, project, info, corpus_bucket_name, quarantine_bucket_name,
         for key, value in sorted(six.iteritems(additional_vars)):
           job.environment_string += ('{} = {}\n'.format(
               key,
-              str(value).encode('unicode-escape')))
+              str(value).encode('unicode-escape').decode('utf-8')))
```

Comment:
In python3 the old version would return a bytes, so when formatting we'll always end up with b'<whatever literal we had before>'. I think this preserves the behaviors we want, and makes test pass again.